partitura/io/importmusicxml.py

```diff
@@ -1441,16 +1441,48 @@ def handle_tuplets(notations, ongoing, note):
         stop_tuplet_key = ("stop_tuplet", tuplet_number)
 
         if tuplet_type == "start":
+            # Get information about the tuplet if present in the XML
+            tuplet_actual = tuplet_e.find("tuplet-actual")
+            tuplet_normal = tuplet_e.find("tuplet-normal")
+            if tuplet_actual is not None and tuplet_normal is not None:
+                tuplet_actual_notes = int(tuplet_actual.find("tuplet-number").text)
+                tuplet_actual_type = tuplet_actual.find("tuplet-type").text
+                tuplet_normal_notes = int(tuplet_normal.find("tuplet-number").text)
+                tuplet_normal_type = tuplet_normal.find("tuplet-type").text
```

Comment:
Instead of calling `attr.find`, I recommend using the function `get_value_from_attribute`, which is slightly more flexible to cover missing values.

Reply:
Maybe I'm wrong, but doesn't `get_value_from_attribute` only work for attributes of a tag? The `<tuplet>` tag is organised as follow:

```xml
        <notations>
          <tuplet type="start" number="2" bracket="no">
            <tuplet-actual>
              <tuplet-number>3</tuplet-number>
              <tuplet-type>16th</tuplet-type>
              </tuplet-actual>
            <tuplet-normal>
              <tuplet-number>2</tuplet-number>
              <tuplet-type>16th</tuplet-type>
              </tuplet-normal>
            </tuplet>
          </notations>
```

And `<tuplet-actual/normal>` are not attributes of `<tuplet>` but rather subelements, hence the `.find` method

Reply:
Yes you are right, I meant to write: `get_value_from_tag`

Reply:
Oh yes I missed this one!